docs/DeveloperGuide.md

```diff
@@ -623,12 +623,18 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `HealthContact` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: UC-01 - Adding a patient**
+**Use case**: UC-01 - A new patient makes an appointment
```

Comment:
may change to "a new patient calls to make an appointment"

Reply:
stated in step 1 alrdy